app/views/clients/_service_summary.haml

```diff
@@ -1,7 +1,7 @@
 - if can_view_confidential_project_names?
-  - seen_before = ( projects = @client.last_projects_served_by(include_confidential_names: true).select(&:present?) ).any?
+  - seen_before = ( projects = @client.last_intentional_contacts(include_confidential_names: true).select(&:present?) ).any?
 - else
-  - seen_before = ( projects = @client.last_projects_served_by.map(&:presence).compact ).any?
+  - seen_before = ( projects = @client.last_intentional_contacts.map(&:presence).compact ).any?
```

Comment:
I could be wrong, but this feels like it could just be:
```
seen_before = ( projects = @client.last_intentional_contacts(include_confidential_names: can_view_confidential_project_names?).select(&:present?) ).any?
```